style.css

```diff
@@ -0,0 +1,136 @@
+body{
+    font-family: "Roboto" , sans-serif;
+    background: #ffffff;
+    width: 100%;
+    height: 100vh;
+    margin: 0;
+    color: #000000;
+    font-size: 15px;
+}
+.container {
+    width: 50%;
+    margin: 0 auto;
+    padding: 10px;
+}
+.items {
+    padding: 8px 48px;
+    margin: 8px;
+    border: 1px solid gray;
+    border-radius: 10px;
+}
+input[type="text"]{
+    outline: none;
+    width: 40%;
+    padding: 8px 25px;
+    border: 0px;
+    border-bottom: 1px solid #DDD;
+    margin: 15px;
+    font-size: 16px;
+}
+.items {
+    text-align:left;
+    border: 1px solid #DDD;
+    font-size: 15px;
+    font-family: "Roboto" , sans-serif;
+    font-weight: normal;
+    color: gray;
+}
+.items input{
+    margin: 0px 0px 0px 0px;
+    position: relative;
+    float:right;
+}
+.close {
+    position: absolute;
+    right: 160px;
+    top: 35px;
+    font-weight: bold;
+}
+.close {
+    background-color: white;
+    color: gray;
+    border: none;
+    text-align: center;
+    font-size: 15px;
+}
+.close:hover {
+    color: rgb(66, 65, 65);
+    color: black;
+    cursor: pointer;
+    font-size: small;
+}
+.items input::before {
+    position: absolute;
```

Comment:
This is a good idea! Using the pseudo elements to get them over the original input, nice!

Another solution is to hide the input and use your own customized styles to show checkmarks, and let the label do the check for you if you added the `for` attribute to it.

I'm not saying that the idea you did is wrong, it is really great and I'm happy with it. Just saying that you can have another option of doing this.